elodie/media/media.py

```diff
@@ -207,6 +211,22 @@ def set_date_taken(self, time):
         self.reset_cache()
         return status
 
+    def set_address(self, location):
+        if(not self.is_valid()):
+            return None
+
+        print('DEBUG - location = %s' % location)
+        if (location is not None):
+            tags = {self.country_key: location['country'],
+                    self.state_key: location['state'],
+                    self.city_key: location['city']
+                    }
+
+        status = self.__set_tags(tags)
```

Comment:
Again, not sure I concur... If we are setting the value, shouldn't we be overwriting existing tags? If I can never overwrite a tag, then I can't "fix one" that's wrong.

Maybe I misunderstand.